examples/widgets/hover_behavior.py

```diff
@@ -0,0 +1,188 @@
+from kivy.uix.recycleview.views import RecycleDataViewBehavior
+
+from kivy.uix.behaviors.hover import HoverBehavior
+
+Builder.load_string("""
+#:import MotionCollideBehavior kivy.uix.behaviors.motion.MotionCollideBehavior
+
+<HoverButton>:
+    canvas.before:
+        Color:
+            rgba: self.background_color
+        Rectangle:
+            pos: self.pos
+            size: self.size
+
+<HoverRecycleLabel>:
+    size_hint_y: None
+    height: 100
+    halign: 'center'
+    valign: 'middle'
+    text_size: self.size
+    canvas.before:
+        Color:
+            rgba: self.bg_color
+        Rectangle:
+            pos: self.pos
+            size: self.size
+
+<SimpleHoverWidget>:
+    canvas.before:
+        Color:
+            rgba: self.background_color
+        Rectangle:
+            pos: self.pos
+            size: self.size
+
+<HoverOpaqueButton@MotionBlockBehavior+Button>:
+    opacity: 0.75
+
+<HoverRecycleView@MotionCollideBehavior+RecycleView>:
+    viewclass: 'HoverRecycleLabel'
+    RecycleBoxLayout:
+        default_size: None, 100
+        default_size_hint: 1, None
+        size_hint_y: None
+        height: self.minimum_height
+        orientation: 'vertical'
+        spacing: 15
+
+<SimpleScreen>:
+    BoxLayout:
+        orientation: 'vertical'
+        padding: 10
+        spacing: 15
+
+        Label:
+            text: 'Simple Hover Test'
+            size_hint_y: 0.1
+            font_size: '20sp'
+
+        SimpleHoverWidget:
+            size_hint: 0.5, 0.8
+            pos_hint: {'center_x': 0.5}
+
+        Button:
+            text: 'Switch to Complex Demo'
+            size_hint_y: 0.1
+            on_press: root.manager.current = 'complex'
+
+<ComplexScreen>:
+    BoxLayout:
+        orientation: 'vertical'
+        padding: 10
+        spacing: 15
+
+        Label:
+            text: 'Complex Hover Demo (RecycleView)'
+            size_hint_y: 0.1
+            font_size: '20sp'
+
+        BoxLayout:
+            spacing: 25
+            size_hint_y: 0.2
+
+            HoverButton:
+                text: 'Button 1'
+            HoverButton:
+                text: 'Button 2'
+            HoverButton:
+                text: 'Button 3'
+
+        HoverRecycleView:
+            id: rv
+            size_hint_y: 0.6
+
+    HoverOpaqueButton:
+        text: 'Switch to Simple Demo'
+        size_hint_y: 0.2
+        on_press: root.manager.current = 'simple'
+""")
+
+
+class HoverButton(HoverBehavior, ButtonBehavior, Label):
+    background_color = ColorProperty([0.2, 0.2, 0.2, 1])
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.original_color = self.background_color[:]
+
+    def on_state(self, _, state):
+        if state == "down":
+            self.background_color = [0.8, 0.8, 0, 1]
+        else:
+            self.background_color = (
+                [0.8, 0, 0.8, 1] if self.hovered else [0.2, 0.2, 0.2, 1]
+            )
+
+    def on_hover_enter(self, me):
```

Comment:
Generally you don't need to override this method to handle a hover event, unless you want to inspect `me` itself. Methods `*_enter` and `*_leave` are called every time a hover event enters or leaves this widget which doesn't have to be a hover from the mouse, but from any hover event (laser pointer or anything dispatching hover events).

To just know if this widget is hovered, you only need to "look at" `hovered` property.